opossum-tool/OpossumUI
# feat: improve package info display

### Summary of changes

- split attribution details into package coordinates and legal information
- redesign action buttons at bottom of attribution column to prevent them from overlapping content while allowing vertical scrolling in the attribution details
- make attribution column actions, especially replacing attributions, clearer
- introduce auditing options and labels at the top of the attribution column to highlight particular states of a signal or attribution
- allow user to set it to an arbitrary value on a scale from 1 to 5 for confidence
- rearrange package coordinate input fields to display most important fields first
- make first-party checkbox a toggle button between first and third party
- simplify multi-line text areas to not affect each other's display state

![image](https://github.com/opossum-tool/OpossumUI/assets/46091730/00dda910-dffc-4eb1-9a27-89bb88957d39)

![image](https://github.com/opossum-tool/OpossumUI/assets/46091730/e928f5fd-be88-43cb-af63-edfa0141c4c0)

![image](https://github.com/opossum-tool/OpossumUI/assets/46091730/eb9e386a-1615-4873-a0ce-b1ef6a8db5b5)

![image](https://github.com/opossum-tool/OpossumUI/assets/46091730/a31c125f-a6e3-4dc1-94dc-76dc419490c7)

![image](https://github.com/opossum-tool/OpossumUI/assets/46091730/3b2d79b9-9346-4688-b736-c53f6ef99cf4)

### Context and reason for change

Closes #2357, #1559, #1572, #2267.

### How can the changes be tested

- Check the attribution column in audit view, attribution view, and the edit attribution popup in report view.
- Create first and third party attributions.
- Assign (multiple) auditing options, including modification of confidence.
- Replace attributions.
